common.go

```diff
@@ -1,4 +1,42 @@
 package sdk
 
+import (
+	"fmt"
+
+	"google.golang.org/grpc"
+)
+
 const NativeBin = "/opt/driver/bin/native"
 const NativeBinTest = "/opt/driver/src/build/native"
+
+const (
+	// MaxMsgSizeCLIName is name of the CLI flag to set max msg size.
+	MaxMsgSizeCLIName = "grpc-max-message-size"
+	// MaxMsgSizeCLIDesc is description for the CLI flag to set max msg size.
+	MaxMsgSizeCLIDesc = "max. message size to send/receive to/from clients (in MB)"
+
+	// DefaulGRPCMaxMsgSizeMb is maximum msg size for gRPC in Mb.
```

Comment:
nit: please be consistent about "Mb" vs "MB". "Mb" usually means "megabits", whereas here it appears to be a capitalization wart maybe(?)

Ideally: "MiB" if we really mean powers of 2.

Reply:
Relatedly: For the gRPC request/reply? Or the whole size including the envelope?

Reply:
Was under impression that "megabits" would be "Mbps" for the thoughtput. 
A [quick](https://github.com/grpc/grpc/commit/61aed92b8262b3a7fa86394cc6b89987ada85228#diff-9d7f390c5994e33fc8848e879931b239R104) [search](https://github.com/grpc/grpc/blob/618a3f561d4a93f263cca23abad086ed8f4d5e86/test/core/compression/message_compress_test.cc#L123) reveals that `MB` seems to be a preferred option here. 👍